tracer/src/Datadog.Trace.Tools.dd_dotnet/Datadog.Trace.Tools.dd_dotnet.csproj

```diff
@@ -10,6 +10,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <RootNamespace>Datadog.Trace.Tools.dd_dotnet</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
```

Comment:
😕 I don't know why this would be required, but I don't think we would want this?

Reply:
Without this build for DDTool fails on MacOS.

Reply:
> Without this build for DDTool fails on MacOS

Yes, it should - Native AOT isn't supported on macOS yet (by .NET) 🙂